bee-message/src/payload/transaction/essence/regular.rs

```diff
@@ -19,6 +19,7 @@ use alloc::vec::Vec;
 pub struct RegularTransactionEssenceBuilder {
     network_id: u64,
     inputs: Vec<Input>,
+    inputs_commitment: [u8; 32],
```

Comment:
It's before the inputs https://github.com/iotaledger/tips/pull/40/commits/1632afd81ddffe564a29f00bdd2585c1af8e3e39

Reply:
https://github.com/iotaledger/tips/pull/40/commits/574bcdcdc8a113d878a88daac3fccb03006d85ab